walkthroughs/week-4-libraries/maps/examples/hello-world/src/main/webapp/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <title>Hello Google Maps</title>
-  <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY"></script>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDjXCSPI4yzByHxDXafuG1ScI5opZmOAyg"></script>
```

Comment:
you shouldn't commit your API key. people/bots are known to trawl through repos looking for them so they can abuse them and rack up bills in your name.